### DISCUSSION TITLE
Compatibility (or issues) with Thunderbolt 4 Docks?

### DISCUSSION THREAD

**Discussion by fabianschulz** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1207] [Q&A] [answered]
Is there any experience with AlDente compatibility of Thunderbolt 4 Docks in general (I am using a MacBook Air M1) and specifically with the Caldigit TS4 Dock? The FAQ on the website only mentions possible issues with Thunderbolt **3** Docks:

_Unfortunately, users have reported that some third-party Thunderbolt 3 docks are incompatible with some of the features of AlDente. These reports mainly concern but are not limited to docks from the brand OWC. Some of the incompatibilities result in disconnects while using Discharge._

I would like to know if any users have reported issues with Thunderbolt 4 Docks using AlDente Pro. 

Thanks!

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @fabianschulz,

We have been testing a Thunderbolt 4 dock with AlDente for a while now and it is working without any issues with the newest version AlDente 1.26.1.

Best,
Matthias

**Comment by fabianschulz** [comment]
Sounds good, thank you Matthias for clarifying!

**Comment by coca7** [comment]
I have a OWC thunderbolt 4 dock and after using the free version of Al Dente for about a week my dock bricked.  I noticed it on a day that I wanted to increase the charging for 80 to 100 and later I noticed it hadn't charged.  Then I realized the hub was dead and would not power up.  It could just be the power supply because it was making funny noises, but since OWC docks were mentioned I thought I would report this.
